lib/Serialization/ModuleFile.cpp

```diff
@@ -2013,6 +2013,16 @@ void ModuleFile::getTopLevelDecls(SmallVectorImpl<Decl *> &results) {
   }
 }
 
+void ModuleFile::getPrecedenceGroups(
```

Comment:
Please add a pretty stack trace to the beginning of the function like we do in e.g. `getTopLevelDecls` so that if deserialization crashes we see this information.
```
PrettyStackTraceModuleFile stackEntry(*this);
```